ui/00_message/common/spot_name.xml

```diff
@@ -2252,6 +2293,7 @@
   <message>
     <key>SPOT_NAME_978</key>
     <original>侵鬼の洞穴　最深部</original>
+    <translation>Deepest part of teh Invading Demons Den</translation>
```

Comment:
Please look at the rest of the file to see how location names like this one are formatted.